packages/flutter_tools/lib/src/android/android_sdk.dart

```diff
@@ -229,6 +232,10 @@ class AndroidSdkVersion implements Comparable<AndroidSdkVersion> {
   String get apksignerPath => getBuildToolsPath('apksigner');
 
   List<String> validateSdkWellFormed({bool requireApkSigner = true}) {
+    if (buildToolsVersion.major < minimumAndroidSdkVersion) {
```

Comment:
I don't think this is entirely right. The `Android SDK` and `Android SDK Build-Tools` are separate installs, are versioned independently, and can be used in different versions:
https://developer.android.com/studio/command-line/index.html

So AFAIK would be better if we had two different min variables,  `minimumAndroidSdkVersion` and `minimumAndroidBuildToolsVersion`. Initially these can both be set to 25.